sdk/react/widgets/ToggleSwitch.js

```diff
@@ -24,7 +25,18 @@ var ToggleSwitch = React.createClass({
     switchOnText: React.PropTypes.string,
     switchOffText: React.PropTypes.string,
     onValueChanged: React.PropTypes.func,
-    config: React.PropTypes.object
+    config: React.PropTypes.object,
+    onTintColor: React.PropTypes.string,
+    tintColor: React.PropTypes.string,
+    thumbTintColor: React.PropTypes.string,
+  },
+
+  getDefaultProps: function() {
+    return {
+      onTintColor: '#498DFC',
```

Comment:
as in.. someone needs to create a new <ToggleSwitch>  Who does this, and how does that class derive those color values?